### PR TITLE
Fix personalizations dynamic bug

### PR DESCRIPTION
**Pull request by burantino**
Fixed two issues:
1. Bug with incorrect dropping of `dynamicTemplateData` from personalization data
2. Correct type for `dynamicTemplateData`

### CODE REVIEW

File: packages/helpers/classes/personalization.d.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export interface PersonalizationData {
   subject?: string;
   headers?: { [key: string]: string };
   substitutions?: { [key: string]: string };
-  dynamicTemplateData?: { [key: string]: string; };
+  dynamicTemplateData?: { [key: string]: any; };
```
Comment (thinkingserious) [inline packages/helpers/classes/personalization.d.ts]: This works, but there are other areas we need to change, see [this PR](https://github.com/sendgrid/sendgrid-nodejs/pull/868) + [line 115](https://github.com/sendgrid/sendgrid-nodejs/blob/ab7dc11bf3ad597321ecbdd59f1480998d3f492c/packages/helpers/classes/personalization.d.ts#L115)

Comment (burantino) [reply packages/helpers/classes/personalization.d.ts]: true, let me fix it as well

Comment (thinkingserious) [reply packages/helpers/classes/personalization.d.ts]: How about here? https://github.com/sendgrid/sendgrid-nodejs/blob/ab7dc11bf3ad597321ecbdd59f1480998d3f492c/packages/helpers/classes/personalization.d.ts#L115

Comment (burantino) [reply packages/helpers/classes/personalization.d.ts]: i've fixed it next commit 👇

**Comment by thinkingserious** [comment]
Hello @burantino,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**